201-machine-learning-encrypted-workspace/azuredeploy.json

```diff
@@ -0,0 +1,177 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "Specifies the name of the Azure Machine Learning workspace."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "allowedValues": [
+        "eastus",
+        "eastus2",
+        "southcentralus",
+        "southeastasia",
+        "westcentralus",
+        "westeurope",
+        "westus2"
+      ],
+      "metadata": {
+        "description": "Specifies the location for all resources."
+      }
+    },
+    "sku":{
+      "type": "string",
+      "defaultValue": "basic",
+      "allowedValues": [
+        "basic",
+        "enterprise"
+      ],
+      "metadata": {
+        "description": "Specifies the sku, also referred to as 'edition' of the Azure Machine Learning workspace."
+      }
+    },
+    "hbi_workspace":{
+      "type": "string",
+      "defaultValue": "false",
+      "allowedValues": [
+        "false",
+        "true"
+      ],
+      "metadata": {
+        "description": "Specifies that the Azure Machine Learning workspace holds highly confidential data."
+      }
+    },
+    "encryption_status":{
+      "type": "string",
+      "defaultValue": "Disabled",
```

Comment:
set this to enabled since that's what the sample is about

Reply:
Enabling this means that we also need to create a Cosmos DB instance during automation. And this requires customer-managed keys be enabled for Cosmos DB. Currently, this is something you have to request be enabled on your subscription (https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-setup-cmk). 

Is this something that's available on the sub used to validate templates? If so, I can work on wiring up the cosmos db creation, etc. so we can enable this.

Reply:
I sent email to enable this in the subs

Reply:
Thanks. I've flagged these as enabled also by default.